pkgs/development/libraries/libdicom/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub, uthash, meson, ninja, cmake, check, pkg-config }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "libdicom";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "ImagingDataCommons";
+    repo = "libdicom";
+    rev = "v${finalAttrs.version}";
+    sha256 = "9n0Gp9+fmTM/shgWC8zpwt1pic9BrvDubOt7f+ZDMeE=";
+  };
+
+  buildInputs = [ uthash ];
+
+  nativeBuildInputs = [ meson ninja cmake check pkg-config ];
```

I am not entirely confident in meson packaging, but this seems like something you'd need while building tests?

```suggestion
  nativeBuildInputs = [ meson ninja cmake pkg-config ];
  
  nativeCheckInputs = [ check ];
```

---

I tested your change but it seems it's required for the configure phase to not fail. Maybe some other nix tricks are required to compile and run tests. For now I opted to to simply remove the check dependency and disable the tests build.